Modules/Core/Common/include/itkAutoPointer.h

```diff
@@ -73,7 +73,11 @@ class AutoPointer
   ~AutoPointer() { this->Reset(); }
 
   /** Overload operator ->.  */
-  ObjectType * operator->() const { return m_Pointer; }
+  ObjectType *
+  operator->() const
+  {
+    return m_Pointer;
+  }
```

Comment:
Interestingly, here it goes the other way (compared to the case of the `m_LastPipelineMTime = 0;` statement). I like having the `return m_Pointer;` statement on its own line.  👍